InriaMuseum/README.md

```diff
@@ -0,0 +1,10 @@
+# Inria Museum
+
+This part is destined to the Inria museum for the exposition. 
+In this part you will find a web interface to launch on the Raspberry Pi.
+
+This interface is coded using : Python, bottle, HTML, SVG & Javascript
+
+STEP 1 : Launch the python script (InriaMuseum.py)
+STEP 2 : Go on a navigator and connect to  http://localhost:8080/ 
+STEP 3 : Connect to Manager fourth port
```

Comment:
Is this accurate? I don’t think your code connects to anything. If so, please remove